docker/Dockerfile

```diff
@@ -31,32 +19,28 @@ RUN \
 # But in our setup, it's sorted out by ide-probe instead (org.virtuslab.ideprobe.ide.intellij.IntellijPrivacyPolicy).
 
 # Tools necessary to run non-headless UI tests in the screen-less environment of CI
-RUN \
-  apt-get update; \
-  apt-get install --no-install-recommends -y libxtst6 xvfb xauth x11-apps imagemagick zip; \
-  `# libraries needed by JCEF` \
```

Comment:
@LukaszKontowski as discussed, I've dropped support for [Java Chromium Embedded Framework](https://github.com/chromiumembedded/java-cef)

Reply:
(as we already for some time no longer use a JetBrains Runtime with JCEF for UI tests)